src/pudl_archiver/metadata/sources.py

```diff
@@ -416,4 +416,40 @@
         "license_pudl": LICENSES["cc-by-4.0"],
         "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
     },
+    "nrelss": {
+        "title": "NREL Standard Scenarios",
+        "path": "https://www.nrel.gov/analysis/standard-scenarios.html",
+        "description": (
+            "NREL's Standard Scenarios are a suite of forward-looking scenarios of the U.S."
+            "power sector that are updated annually to support and inform energy analysis."
+            "The Standard Scenarios are simulated using the Regional Energy Deployment System"
+            "and Distributed Generation Market Demand Model capacity expansion models and are"
+            "updated each year to provide timely information regarding power sector evolution."
+            "The scenarios have been designed to capture a range of possible power system"
+            "futures and consider a variety of factors from high vehicle electrification to"
+            "major cost declines for electricity generation technologies (e.g., using cost"
+            "inputs from the Annual Technology Baseline)."
+            "For select scenarios, the models are run using the PLEXOS software and the"
+            "Cambium tool that assembles structured data sets of hourly cost, emissions, and"
+            "operational data for modeled futures. Results are available using the Scenario"
+            "Viewer and Data Downloader."
+        ),
+        "source_file_dict": {
+            "source_format": "CSV",
+        },
+        "working_partitions": {
+            "years": list(range(2016, 2025)),
```

Comment:
There seem to be two geo levels available as well, with one set of files for `Nation` and one set for `States`. Should we include that as a partition, or leave it at years?

Reply:
partitions are meant to be able to uniquely grab one file from the archive. so if the plan is to put both the nation and states for one year into one zip file then you can keep only the years are the partitions. without knowing how the archiver will work or how the files are structured my inclination would be to squish both nation and states into one annual zip.

Reply:
Resolved offline: since not every report year has the same location options, and particularly because 2020 only has "ALL" as a location option, we're not going to partition by geography